py-polars/polars/selectors.py

```diff
@@ -491,34 +512,79 @@ def datetime(time_unit: TimeUnit | None = None) -> Expr:
     ...         ],
     ...         "dt": [date(1999, 12, 31), date(2010, 7, 5)],
     ...     },
-    ...     schema_overrides={"tstamp": pl.Datetime("ns"), "dtime": pl.Datetime("ms")},
+    ...     schema_overrides={
+    ...         "tstamp_utc": pl.Datetime("ns", "UTC"),
+    ...         "tstamp": pl.Datetime("ns"),
+    ...     },
     ... )
 
     Select all datetime columns:
 
     >>> df.select(cs.datetime())
-    shape: (2, 2)
-    ┌─────────────────────────┬─────────────────────────┐
-    │ tstamp                  ┆ dtime                   │
-    │ ---                     ┆ ---                     │
-    │ datetime[ns]            ┆ datetime[ms]            │
-    ╞═════════════════════════╪═════════════════════════╡
-    │ 2000-11-20 18:12:16.600 ┆ 2010-10-10 10:25:30.987 │
-    │ 2020-10-30 10:20:25.123 ┆ 2024-12-31 20:30:45.400 │
-    └─────────────────────────┴─────────────────────────┘
+    shape: (2, 3)
+    ┌────────────────────────────────┬─────────────────────────┬────────────────────────────┐
+    │ tstamp_utc                     ┆ tstamp                  ┆ dtime                      │
+    │ ---                            ┆ ---                     ┆ ---                        │
+    │ datetime[ns, UTC]              ┆ datetime[ns]            ┆ datetime[μs]               │
+    ╞════════════════════════════════╪═════════════════════════╪════════════════════════════╡
+    │ 2023-04-10 12:14:16.999666 UTC ┆ 2000-11-20 18:12:16.600 ┆ 2010-10-10 10:25:30.987    │
+    │ 2025-08-25 14:18:22.666333 UTC ┆ 2020-10-30 10:20:25.123 ┆ 2024-12-31 20:30:45.400500 │
+    └────────────────────────────────┴─────────────────────────┴────────────────────────────┘
 
     Select all datetime columns that have 'ns' precision:
 
     >>> df.select(cs.datetime("ns"))
+    shape: (2, 2)
+    ┌────────────────────────────────┬─────────────────────────┐
+    │ tstamp_utc                     ┆ tstamp                  │
+    │ ---                            ┆ ---                     │
+    │ datetime[ns, UTC]              ┆ datetime[ns]            │
+    ╞════════════════════════════════╪═════════════════════════╡
+    │ 2023-04-10 12:14:16.999666 UTC ┆ 2000-11-20 18:12:16.600 │
+    │ 2025-08-25 14:18:22.666333 UTC ┆ 2020-10-30 10:20:25.123 │
+    └────────────────────────────────┴─────────────────────────┘
+
+    Select all datetime columns that have *any* timezone:
+
+    >>> df.select(cs.datetime(time_zone="*"))
```

Comment:
looks like the output here is the same as the output below ("Select all datetime columns that have a *specific* timezone:"), just wondering if it's better to make one of the original columns tz-aware but with a different time zone, so the example's clearer?

Reply:
Done 👍